### PR TITLE
ODC-7319: correcting - missing package tag across gherkin files

### PR DESCRIPTION
**Pull request by The-Anton**
opening a PR for correcting missing package tag across gherkin files.

Jira Id:

- Story: [ODC-7319](https://issues.redhat.com/browse/ODC-7319)

### CODE REVIEW

File: frontend/packages/webterminal-plugin/integration-tests/features/web-terminal/web-teminal-basic.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@web-terminal
+@web-terminal @dev-console
```
Comment (sanketpathak) [inline frontend/packages/webterminal-plugin/integration-tests/features/web-terminal/web-teminal-basic.feature:1]: This is not part of devconsole package anymore

File: frontend/packages/dev-console/integration-tests/features/config-maps/config-maps-dev-perspective.feature [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@config-maps
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/config-maps/config-maps-dev-perspective.feature:1]: Do we also need `@dev-console` label here?

File: frontend/packages/webterminal-plugin/integration-tests/features/web-terminal/web-teminal-basic.feature (hunk 1 above)
Comment (The-Anton) [reply frontend/packages/webterminal-plugin/integration-tests/features/web-terminal/web-teminal-basic.feature]: Sure, removed the tags!

File: frontend/packages/dev-console/integration-tests/features/config-maps/config-maps-dev-perspective.feature (hunk 2 above)
Comment (The-Anton) [reply frontend/packages/dev-console/integration-tests/features/config-maps/config-maps-dev-perspective.feature]: Yes, I have added the tags!

**Comment by sanketpathak** [comment]
PR only affects our e2e tests 
/label docs-approved
/label px-approved
/label qe-approved

**Comment by sanketpathak** [comment]
/retitile ODC-7319: correcting - missing package tag across gherkin files

**Comment by sanketpathak** [comment]
/retitle ODC-7319: correcting - missing package tag across gherkin files

**Pull request merged**